From 278b2cbc2416d55e9c0a6e5932b4e02a764f56f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:16:46 +0800
Subject: [PATCH] Revert "Staging: android: timed_gpio: Properly discard
 invalid timeout values."

This reverts commit a7f5cb901df253cb3aed7a369298a95703cc5995.
---
 drivers/staging/android/timed_output.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/staging/android/timed_output.c b/drivers/staging/android/timed_output.c
index f373422308e0..62e79180421b 100644
--- a/drivers/staging/android/timed_output.c
+++ b/drivers/staging/android/timed_output.c
@@ -41,9 +41,7 @@ static ssize_t enable_store(
 	struct timed_output_dev *tdev = dev_get_drvdata(dev);
 	int value;
 
-	if (sscanf(buf, "%d", &value) != 1)
-		return -EINVAL;
-
+	sscanf(buf, "%d", &value);
 	tdev->enable(tdev, value);
 
 	return size;
-- 
2.35.3

